.gitpod.yml

```
@@ -12,20 +12,21 @@ tasks:
       go mod tidy
       mkdir -p web/dist
       touch web/dist/index.html
+    command: |
       echo WOODPECKER_GITEA_URL=https://3000-${GITPOD_WORKSPACE_ID}.${GITPOD_WORKSPACE_CLUSTER_HOST} >> .env
       echo WOODPECKER_DEV_OAUTH_HOST=https://8000-${GITPOD_WORKSPACE_ID}.${GITPOD_WORKSPACE_CLUSTER_HOST} >> .env
       gp sync-await gitea
+      go run github.com/woodpecker-ci/woodpecker/cmd/server
       gp sync-done woodpecker-server
```

```suggestion
      gp sync-done woodpecker-server
      go run github.com/woodpecker-ci/woodpecker/cmd/server
```

I think `go run` will block so we would never get to `gp sync-done`